lib/gnat.ex

```diff
@@ -88,30 +88,30 @@ defmodule Gnat do
   * `tls_available` - If the server can use TLS
   """
   @type server_info :: %{
-    :client_id => non_neg_integer(),
-    :client_ip => binary(),
-    optional(:ip) => binary(),
-    optional(:cluster) => binary(),
-    optional(:cluster_dynamic) => boolean(),
-    optional(:connect_urls) => list(binary()),
-    optional(:ws_connect_urls) => list(binary()),
-    optional(:git_commit) => binary(),
-    :go => binary(),
-    :headers => boolean(),
-    :host => binary(),
-    optional(:jetstream) => binary(),
-    :max_payload => integer(),
-    :port => non_neg_integer(),
-    :proto => integer(),
-    :server_id => binary(),
-    :server_name => binary(),
-    :version => binary(),
-    optional(:ldm) => boolean(),
-    optional(:tls_verify) => boolean(),
-    optional(:tls_available) => boolean(),
-    optional(:tls_required) => boolean(),
-    optional(:auth_required) => boolean(),
-  }
+          :client_id => non_neg_integer(),
```

Comment:
Looks like my formatter changed this as well. If you'd like, I can split this out into a different PR

Reply:
I thought I had a formatter check built into the CI process, but maybe I'm misremembering. If this is the standard formatter settings for current elixir versions then we should just apply it now.

Reply:
TBH i'm not sure where the difference is coming from